plugin/src/main/java/com/appunite/firebasetestlabplugin/FirebaseTestLabPlugin.kt

```diff
@@ -367,11 +356,30 @@ internal class FirebaseTestLabPlugin : Plugin<Project> {
             }
         }
     }
+
+    private fun addExecuteAndDownload(name: String, downloader: CloudTestResultDownloader?, cleanTask: String, taskClosure: Closure<Any?>): Task {
```

Comment:
I think we should stick to `taskClosure: Closure<Task>` I'm not big fan of giving to much space to make a mistake. We can always expand it if needed